cockroachcloud/cloud-api.md

```diff
@@ -6,94 +6,15 @@ toc: true
 
 The Cloud API is a [REST interface](https://en.wikipedia.org/wiki/Representational_state_transfer) that allows users programmatic access to manage the lifecycle of clusters within their organization.
 
-{{site.data.alerts.callout_danger}}
-This page describes an early access, experimental version of the Cloud API. The interface and output is subject to change, and there may be bugs.
```

Comment:
**Suggested wording:** 

This page describes the private Early Access version of the Cloud API. It's for **experimental use only**. The interface and output are subject to breaking changes. We would be grateful for your feedback - please email Mark Soper [contact info](https://github.com/marksoper) to request access.

Reply:
this looks good. @marksoper I would suggest maybe having an external facing email alias created for this with more team members on it. This way we can give more visibility into folks reaching out and you can still be the owner of the email group.

Reply:
Like the email alias idea. Also, I think this include needs to be crafted so it's not about an entire page. It could get included for a section of a page, for example.

Reply:
Should this PR wait until we have the alias ready? I will update the new Cloud-specific private/experimental warning with the contact info.

Reply:
I agree, an email group alias makes more sense.

Reply:
@marksoper, can we quickly set up an email alias so we can get this published?

In the meantime, @ianjevans and @marksoper, can you work out how to extend the warning to guide readers on how to use that email alias?

Reply:
I had helpdesk create a new email group, earlyaccess@cockroachlabs.com. @marksoper is the current member, and he can add others in groups.google.com.